makejdk.sh

```diff
@@ -215,6 +218,10 @@ else
   $WORKING_DIR/sbin/build.sh $WORKING_DIR $TARGET_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME $JVM_VARIANT
 
   if [[ ! -z $JTREG ]]; then
-    $WORKING_DIR/sbin/jtreg.sh $WORKING_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME
+    if [[ ! -z $JTREG_TEST_SUBSETS ]]; then
+      $WORKING_DIR/sbin/jtreg.sh $WORKING_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME $JTREG_TEST_SUBSETS
```

Comment:
Do we have a separate issue tracking why langtools is failing?

Ideally we'd default to jdk_all for release builds and yeah probably jdk_core for nightly builds.  So perhaps this script should default to jdk_core (a decent speed/coverage) compromise, but we can override that in our various build scripts.

IIRC jdk_all will also perform UI tests (Swing/AWT) - so will expose other missing deps that our build boxes may need.